modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -124,7 +124,7 @@ If the store is seeded with `seedURI`, `existingDataSeedInstance` or `existingDa
 
 [NOTE]
 ====
-The `existingData`, `existingDataSeedInstance`, `existingDataSeedServer`, `seedURI`, `seedConfig`, and `seedCredentials` options cannot be combined with the `OR REPLACE` part of this command.
+The `existingDataSeedInstance` and `existingDataSeedServer` options cannot be combined with the `OR REPLACE` part of this command.
```

Comment:
You're removing a few options here. Does this mean that starting from 2025.01, `existingData`, `seedURI`, `seedConfig`, and `seedCredentials` options **can** be combined with the `OR REPLACE` part?

Reply:
Yes exactly.
This change is not dependent on the cypher version.